src-db/database/model/triggers/C_ORDERLINETAX_TRG.xml

```diff
@@ -163,11 +163,16 @@ BEGIN
       WHERE C_Order_ID = :OLD.C_Order_ID
       AND C_Tax_ID = :OLD.C_Tax_ID;
     END IF;
-    DELETE FROM C_ORDERTAX 
-    WHERE C_ORDER_ID = :OLD.C_Order_ID
-    AND C_TAX_ID = :OLD.C_TAX_ID
-    AND ROUND(TAXBASEAMT, v_Prec) = 0
-    AND TAXAMT = 0;
+	DELETE FROM C_ORDERTAX 
```

Comment:
Before the XML had spaces and now it has tabs, check it and try to have it unified (Only tabs or only spaces). The lines before and after the changes have only spaces